docs/src/pages/demos/text-fields/TextFields.js

```diff
@@ -96,6 +96,16 @@ class TextFields extends React.Component {
           autoComplete="current-password"
           margin="normal"
         />
+        <TextField
+          id="read-only-input"
+          label="Read Only"
+          defaultValue="Hello World"
+          className={classes.textField}
+          margin="normal"
+          InputProps={{
```

Comment:
Error: capital-case, use small case for InputProps - > inputProps

Reply:
@jmav Both works. It was made on purpose.

Reply:
Input is the react component, input is the Native element. You can provide props / attributes to both of them. The distinction os covered in the API docs.